src/main/scala/de/retest/guistatemachine/api/impl/serialization/GuiStateMachinGMLSerializer.scala

```diff
@@ -48,12 +48,12 @@ class GuiStateMachinGMLSerializer(guiStateMachine: GuiStateMachine) extends GuiS
 
     allStatesSorted.foreach { x =>
       val fromVertex = x._1
-      val allTransitionsSorted = x._2.getTransitions.toSeq.sortWith(hashCodeComparisonOfTuples)
+      val allTransitionsSorted = x._2.getOutgoingActionTransitions.toSeq.sortWith(hashCodeComparisonOfTuples)
 
       allTransitionsSorted foreach { transition =>
         val actionTransitions = transition._2
         val action = transition._1
-        actionTransitions.to.foreach { toState =>
+        actionTransitions.states.foreach { toState =>
```

Comment:
Since `states` contains "transitions lead to or start from", does this now causes transitions potentially being drawn twice?

Reply:
It uses `getOutgoingActionTransitions` above, so probably no but I will rename the variable.
I will also increase the coverage again.